src/Shared/Components/InvalidYAMLTippy/InvalidYAMLTippyWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import { Tooltip } from '@Common/Tooltip'
+import { InvalidYAMLTippyWrapperProps } from './types'
+import { getInvalidTippyContent } from './utils'
+
+const InvalidYAMLTippy = ({ parsingError, restoreLastSavedYAML, children }: InvalidYAMLTippyWrapperProps) => (
+    <Tooltip
+        alwaysShowTippyOnHover
+        interactive
+        content={getInvalidTippyContent({
+            parsingError,
+            restoreLastSavedYAML,
+        })}
+    >
+        {children}
+    </Tooltip>
+)
+
+const InvalidYAMLTippyWrapper = ({ parsingError, restoreLastSavedYAML, children }: InvalidYAMLTippyWrapperProps) => {
+    if (parsingError) {
+        return (
+            <InvalidYAMLTippy parsingError={parsingError} restoreLastSavedYAML={restoreLastSavedYAML}>
+                {children}
+            </InvalidYAMLTippy>
+        )
+    }
+
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    return <>{children}</>
```

Comment:
```suggestion
    return children
```

since childern can only be ReactElement or JSX.Element